docs/source/basic-usage.rst

```diff
@@ -213,11 +213,12 @@ connection object and start handing it data. For now, let's just see what
 happens as we feed it data.
 
 To make HTTP/2 connections, we need a tool that knows how to speak HTTP/2.
-Most versions of curl in the wild don't, so let's install a Python tool. In
-your Python environment, run ``pip install hyper``. This will install a Python
-command-line HTTP/2 tool called ``hyper``. To confirm that it works, try
-running this command and verifying that the output looks similar to the one
-shown below:
+You can simply use `curl`_ or install a Python tool used throughout this
+tutorial. In your Python environment, run ``pip install hyper`` (Make sure to
+use ``Python < 3.10`` or install it in separate environment). This will
+install a Python command-line HTTP/2 tool called ``hyper``. To confirm that
+it works, try running this command and verifying that the output looks similar
+to the one shown below:
```

Comment:
Since the `hyper` project itself is archived / no longer supported / not maintained, I would suggest to remove the mention of it in this paragraph entirely. `curl` is sufficiently modern nowadays and can be the only mention here.

Reply:
Do you agree with removing `hyper` from this paragraph?